website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -6,37 +6,34 @@ description: >
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
-These include:
-
-* Marking the node as unschedulable, so no further pods can be scheduled there.
-* Evicting all pods other than daemonsets from the node.
-* Terminating the instance from the cloud provider.
-* Deleting the node from the Kubernetes cluster.
+The finalizer blocks deletion of the node object while the Termination Controller cordons and drains the node, before removing the underlying machine. Deprovisioning is triggered by the Deprovisioning Controler, by the user through manual deprovisioning, or through an external system that sends a delete request to the node object.
+
+### Deprovisioning Controller
+Karpenter automatically discovers deprovisionable nodes and spins up replacements when needed. Karpenter deprovisions nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, then Consolidation. Each method varies slightly but they all follow the standard deprovisioning process:
+1. Identify a list of prioritized candidates for the deprovisioning method.
+   1. If there are no deprovisionable nodes, continue to the next deprovisioning method.
+2. For each deprovisionable node, execute a scheduling simulation with the pods on the node to find if any replacement nodes are needed.
+3. Cordon the node(s) to prevent pods from scheduling to it.
+4. Pre-spin any replacement nodes needed as calculated in Step (2), and wait for them to become ready.
+   1. If a replacement node fails to initialize, un-cordon the node(s), and restart from at Step (1).
+5. Delete the node(s) and wait for the termination controller to gracefully shutdown the node(s).
+6. Once the termination controller terminates the node, go back to Step (1), starting at the the first method again.
+
+### Termination Controller
+When a Karpenter node is deleted, the Karpenter finalizer will block deletion and the APIServer will set the `DeletionTimestamp` on the node, allowing Karpenter to gracefully shutdown the node, modeled after [K8s Graceful Node Shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown). Karpenter's graceful shutdown process will:
+1. Cordon the node to prevent pods from scheduling to it.
+2. Begin evicting the pods on the node with the [K8s Eviction API](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/) to respect PDBs, while ignoring all non-daemonset pods and [static pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/). Wait for the node to be fully drained before proceeding to Step (3).
+   1. While waiting, if the underlying instance of the node no longer exists, remove the finalizer to allow the APIServer to delete the node, completing termination.
```

Comment:
```suggestion
   1. While waiting, if the underlying machine for the node no longer exists, remove the finalizer to allow the APIServer to delete the node, completing termination.
```

nit: probably want to start using this lingo and being consistent throughout our docs to get users familiar with the naming